gatsby-restart/src/data/docs/wormhole/payloadcreation.md

```diff
@@ -0,0 +1,312 @@
+---
```

Comment:
Let's name these with a hyphen, such as`payload-creation.md`.
Since the paths are made from the file-name, i think that will have a nicer path

Reply:
Agree. I made these changes